libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/onboard/MapboxOnboardRouter.kt

```diff
@@ -27,10 +30,12 @@ import kotlinx.coroutines.withContext
  * tiles' version, token. Config is provided via [RoutingTilesOptions].
  *
  * @param navigatorNative Native Navigator
+ * @param context application [Context]
  * @param logger interface for logging any events
  */
 class MapboxOnboardRouter(
     private val navigatorNative: MapboxNativeNavigator,
+    private val context: Context,
```

Comment:
This is a property, any chance we could leak the `Context`? I know that should be the `applicationContext` but double-checking here.

Reply:
I'd not worry about that, as you mentioned it's appContex, and this approach we use with  `MapboxOffboardRouter` https://github.com/mapbox/mapbox-navigation-android/blob/07fe4ba304cad3438797ea968932a2f9ec1fff21/libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/offboard/MapboxOffboardRouter.kt#L21-L33